hamcrest/Hamcrest/Type/IsNumeric.php

```diff
@@ -19,10 +19,31 @@ public function __construct()
 
     public function matches($item)
     {
+        if ($this->isHexadecimal($item)) {
+
+            return true;
+        }
+
         return is_numeric($item);
     }
 
     /**
+     * This check is necessary because PHP 7 doesn't recognize hexadecimal string as numeric anymore.
+     *
+     * @param mixed $item
+     * @return boolean
+     */
+    private function isHexadecimal($item)
+    {
+        if (is_string($item) && preg_match('/^0x(.*)$/', $item, $matches )) {
+
```

Comment:
Empty line inside IF statement again. Is there any coding standard you're using to create such IFs?